### ISSUE TITLE
tsclean failed for seasonal time series with only zeros and NAs

### ISSUE THREAD

**Issue by brick01** [robjhyndman/forecast#723]
Hi `tsclean` is unable to fill NAs with zeros for seasonal time series with only zeros and NAs. It has no problem to fill NAs with zeros for non-seasonal time series with only zeros and NAs.

Reproducible example:

```
time_series <- ts(c(0, 0, 0, 0, 0, NA, 0, 0, 0, 0, 0), frequency = 4)
tsclean(time_series)
```

Result:

```
Error in if (strength >= 0.05) xx <- seasadj(fit) : 
  missing value where TRUE/FALSE needed
```

Is this to be expected? Thanks

**Comment by mitchelloharawild** [comment]
I can't reproduce this on the latest version of forecast.
``` r
devtools::load_all(".")
#> Loading forecast
time_series <- ts(c(0, 0, 0, 0, 0, NA, 0, 0, 0, 0, 0), frequency = 4)
tsclean(time_series)
#>   Qtr1 Qtr2 Qtr3 Qtr4
#> 1    0    0    0    0
#> 2    0    0    0    0
#> 3    0    0    0
```

Created on 2018-07-18 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by robjhyndman** [comment]
That works ok for me. Please update your packages and try again.

**Comment by brick01** [comment]
Problem solved by updating forecast package to 8.4. Thank you for the quick reply!
